db/structure.sql

```diff
@@ -367,6 +367,20 @@ CREATE TABLE `community_customizations` (
   KEY `index_community_customizations_on_community_id` (`community_id`) USING BTREE
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
+DROP TABLE IF EXISTS `community_domain_checkers`;
```

Comment:
This is probably added to the PR by mistake as it is related to old migration that was removed from the code. You might need to reset your local databases to avoid getting that back in.